### PR TITLE
Small lumpy fixes

### PR DESCRIPTION
**Pull request by apregier**
I am hoping to run lumpy in a build - putting in some remaining small fixes to try it out.

### CODE REVIEW

**Comment by ebelter** [comment]
+1

File: lib/perl/Genome/Model/Tools/DetectVariants2/Lumpy.pm [hunk 1]
```diff
@@ -17,7 +17,12 @@ class Genome::Model::Tools::DetectVariants2::Lumpy {
             calculate_from => ['_temp_staging_directory'],
             calculate => q{ File::Spec->join($_temp_staging_directory, 'legend.tsv'); },
         },
-    ]
+    ],
+    has_param => [
+        lsf_resource => {
+            default_value => "-M 16000000 -R 'select[mem>16000] rusage[mem=16000]'",
```
Comment (tmooney) [inline lib/perl/Genome/Model/Tools/DetectVariants2/Lumpy.pm]: This should probably include a request for some temp space.

Comment (apregier) [inline lib/perl/Genome/Model/Tools/DetectVariants2/Lumpy.pm]: Yeah, my test build ran into that problem!  I am also going to investigate whether we can use StreamGraph to optimize this a little bit and not write out some of the files.  But we will still need temp space.

**Comment by apregier** [comment]
New versions of lumpy are much easier to run, so I don't think it is worth the effort to get this version working.

**Pull request closed without merging**